### PR TITLE
FastAPI integration docs

### PR DESCRIPTION
**Pull request by leplatrem**
Merge into #75

### CODE REVIEW

File: docs/fastapi.rst [hunk 1]
```diff
@@ -0,0 +1,320 @@
...
+   configuration), see :ref:`fastapi-logging` for more information.
+
+.. _fastapi-config:
+
+Configuration
+-------------
+
+.. epigraph::
+
+   Accept its configuration through environment variables.
```
Comment (grahamalama) [inline docs/fastapi.rst:65]: Does `epigraph` suggest that this should have more formatting that just an indentation?
image

**Review by grahamalama — COMMENTED** [review]
A few inline comments, and this more higher-level thought:

Some of the documentation we've added feels like documentation of FastAPI vs how to integrate Dockerflow's FastAPI extension into a FastAPI app. 

Can we link to FastAPI documentation where appropriate and focus this documentation on aspects that are Dockerflow-specific?

**Comment by grahamalama** [comment]
> This is true about all the rest of the documentation I believe, I aligned it with other integrations actually.

Ahh okay. Well then, I think internal consistency outweighs that feedback I gave. Maybe look for places were we're _missing_ integration docs, but if that spills into FastAPI documentation, so be it.

**Review by grahamalama — APPROVED** [review]

File: docs/fastapi.rst [hunk 2]
```diff
@@ -0,0 +1,320 @@
...
+
+   Must have a JSON version object at /app/version.json.
+
+Dockerflow requires writing a `version object`_ to the file
+``/app/version.json`` as seen from the docker container to be served under
+the URL path ``/__version__``.
+
+To facilitate this python-dockerflow comes with a FastAPI view to read the
+file under path the parent directory of the app root. See the
+:class:`FastAPI API docs <~fastapi.FastAPI>` for more information about the
```
[the remark this answers is not in the document]
Comment (leplatrem) [reply docs/fastapi.rst:140]: Apparently, we have the same behaviour on Flask docs https://python-dockerflow.readthedocs.io/en/main/flask.html#versions

File: docs/fastapi.rst (hunk 1 above)
Comment (leplatrem) [reply docs/fastapi.rst:65]: This is like a note... I'd say that it just has some indentation

**Pull request merged**